Imaginary-tvOSTests/Imaginary_tvOSTests.swift

```diff
@@ -0,0 +1,36 @@
+//
+//  Imaginary_tvOSTests.swift
+//  Imaginary-tvOSTests
+//
+//  Created by Khoa Pham on 22.09.2017.
+//  Copyright © 2017 Hyper Interaktiv AS. All rights reserved.
+//
+
+import XCTest
+@testable import Imaginary_tvOS
+
+class Imaginary_tvOSTests: XCTestCase {
+    
```

Comment:
What's the difference between `Imaginary-tvOSTests` and `Imaginary_tvOSTests` folders?

Reply:
I have no idea. I will clean this up